src/tests/mockData/profile.js

```diff
@@ -0,0 +1,43 @@
+const profile = {
+  user: {
+    username: 'seremnn',
+    email: 'serem@testing.com',
+    token: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6OCwidXNlcm5hbWUiOiJzZXJlbW5uIiwiZXhwIjoxNTU1Mjc0OTgwfQ.DVRlpBQWpgt5mLIwumzgeOS8eHXCMpSHk6XpJaqMPn0',
```

Comment:
@BrianSerem, what is the token being used for on this line, Hope it is a fake token.

Reply:
Yeah, this is a dummy token for testing purposes